pkg/virt-controller/watch/migration/migration.go

```diff
@@ -979,6 +981,12 @@ func (c *Controller) handleTargetPodHandoff(migration *virtv1.VirtualMachineInst
 	if !c.isMigrationPolicyMatched(vmiCopy) {
 		vmiCopy.Status.MigrationState.MigrationConfiguration = clusterMigrationConfigs
 	}
+	// TODO: disable post-copy for volume migration until is properly tested
+	if volmig.IsVolumeMigrating(vmiCopy) &&
+		vmiCopy.Status.MigrationState.MigrationConfiguration != nil {
```

Comment:
Actually should this be: `vmiCopy.Status.MigrationState.MigrationConfiguration != nil && vmiCopy.Status.MigrationState.MigrationConfiguration.AllowPostCopy != nil && *vmiCopy.Status.MigrationState.MigrationConfiguration.AllowPostCopy`